### PR TITLE
create pull from microverse Readme template and join it with my trial project

### PR DESCRIPTION
**Pull request by joffreynk**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ @media only screen and (max-width: 500px) {
+
+} 
+
+
+#survey-form{
+  background-color: rgba(0,0,0,0.5);
+  box-sizing: border-box;
+  padding: 40px;
+}
```
Comment (joffreynk) [inline css/style.css:58]: add one line at the end of each code

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.container {
+    width: 740px;
+}
+
+ @media only screen and (max-width: 500px) {
```
Comment (joffreynk) [inline css/style.css:49]: #changes required

media query is created but not used. remote it

Comment (joffreynk) [inline css/style.css:49]: make sure you put indentation of 2 spaces

File: index.html [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="container">
+      <div class="text-center pt-5 pb-2">
```
Comment (joffreynk) [inline index.html:15]: group the heading info into the heading tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div class="container">
```
Comment (joffreynk) [inline index.html:14]: use <main> instead to apply html best practices

File: index.html [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+              DForum
+            </label>
+          </div>
+          <div class="form-check">
+            <input class="form-check-input" type="checkbox" value="Additional id="opc10">
+            <label class="form-check-label" for="opc10">
+              Additional Courses
+            </label>
+          </div>                                                                      
+        </div>
```
Comment (joffreynk) [inline index.html:142]: put the code in <ul> and replace divs with li

File: index.html [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+            </label>
+          </div>                                                                      
+        </div>  
+        
+        <div class="mb-4">
+          <label class="form-label" for="comments">Any comments or suggestions?</label>
+          <textarea class="form-control" id="comments" rows="5" placeholder="Enter your comments here..."></textarea>
+        </div>
+       
+        <div class="mb-4">
```
Comment (joffreynk) [inline index.html:149]: add the margin button to the container to help your form not stick to the bottom of the page

File: index.html [hunk 7]
```diff
@@ -0,0 +1,157 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
```
Comment (joffreynk) [inline index.html:13]: Unexpected empty lines: they are used for separating sections of code or for readability purposes

**Review by joffreynk — COMMENTED** [review]
#changes required ♻️ 

see below changes and try to figured them out

come back when you have met all them, don`t hesitate to contact me for help or any other question related to this project
se me here:  [@joffreynk](https://github.com/JoffreyNK)
Thanks

**Review by Ismailco — CHANGES_REQUESTED** [review]
Hi @JoffreyNK,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good use of Github inline comments
✔️ Used more than 2 comments

## Required Changes ♻️
- [ ] You don't always phrase the comments as suggestions, please consider using suggestions instead of giving orders, that makes your review more polite
- [ ] You don't always include an explanation for each requested change, kindly include a 'why' to every change that you request
- [ ] Your comments are not always clear  and easy to understand, kindly consider rephrasing your comment's so they give a clear meaning

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Point out that the PR has no description, it's best to always add a description for your PR
- [x] Point out that the linter's are not passing, linters check the code for errors to fix so that the code has best practice 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -0,0 +1,157 @@
...
+      <div class="text-center pt-5 pb-2">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+      </div>
+      <form name="form" method="get" action="#" id="survey-form">
+        <div class="mb-4">
+          <label class="form-label" id="name-label" for="name">Name</label>
+          <input type="text" class="form-control" id="name" placeholder="Enter your name" required>
+        </div>
+
```
Comment (joffreynk) [inline index.html:24]: if you don`t matter, it is best practice to not add unnecessary empty line

File: index.html (hunk 4 above)
Comment (joffreynk) [inline index.html:14]: please, look at the end of this block, it was not closed.it is not good to leave some tags unclosed, they may confuse you when debugging your project in future

File: css/style.css (hunk 1 above)
Comment (joffreynk) [inline css/style.css:58]: according to linters rules you have to end each line with a single empty line, can you add it here, please

File: css/style.css (hunk 2 above)
Comment (joffreynk) [inline css/style.css:49]: media query is not required and was not used, please you can remove it

**Review by joffreynk — COMMENTED** [review]
#New changes required ♻️ 
thanks for solving the previous changes
you are ready go

only below changes remain

File: index.html (hunk 4 above)
Comment (youmari) [reply index.html:14]: @JoffreyNK it would be better if your message were clear and understandable, please try to make your message more clear

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @JoffreyNK , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes

- [ ] please try to update all the comments as the previous asked by explaining why you requiring changes and your messages should be clear

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by joffreynk** [comment]
Can you help me to get this work done? i don't see how I can update a
comment

**Review by eliasecasta — COMMENTED** [review]
Hello @JoffreyNK 👋,

_This is an additional review as a result of the second opinion request._

As I can observe, you applied the requested changes by Ismail by creating new comments with the corrections, this is also acceptable, it is not necessary to edit your previous comments, in this case, both approaches are correct for this project.

Regarding your edition question, you can click on the 3 dots to the right of your comments and an edit option will appear:

image

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please ask for another review in your Dashboard and the next Code Reviewer will mark it as approved right away :)

Keep rocking 🚀!

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @JoffreyNK 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice use of GitHub inline comment 👍 



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joffreynk** [comment]
can you give me an example of how I should explain

**Pull request merged**